source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -138,26 +138,25 @@ def post(self, request, *args, **kwargs):
                     type='account_activation',
                 ).key
 
-                # Setup the url for the activation button in the email
-                activation_url = FRONTEND_SETTINGS['ACTIVATION_URL'].replace(
-                    "{{token}}",
-                    activate_token
-                )
+                # data for email activation
+                merge_data = {
+                    'ACTIVATION_URL': FRONTEND_SETTINGS['ACTIVATION_URL'].replace(
+                        "{{token}}",
+                        activate_token
+                    ),
+                    'CSS_STYLE': render_to_string("css/confirm_sign_up.css")
```

Comment:
This should be included in the "services.py". CSS will always be located at the same place and specifying it in the merge data every time will be redundant.

Reply:
done

Reply:
Je ne parlais que de la ligne concernant le CSS. Le reste du merge data est tout à fait arbitraire et dépend du template utilisé. Il doit donc être possible de le modifier. Toutefois, la clé CSS qui y est présente deviendra vite redondante, au même titre que les template_path. En réalité, c'est le même enjeu que les template_path. Le CSS se trouvera-t-il toujours au même endroit relativement au template? Si oui, est-il pertinent de devoir fournir ce chemin à la fonction utilitaire?